src/service/discovery.js

```diff
@@ -179,14 +180,14 @@ const fetchInitialDiscoveryTopics = async (limit = 25, page = 0) => {
  * @param {Number} page
  * @returns {FetchInitialDiscoveryUsersResponse}
  */
-const fetchInitialDiscoveryUsers = async (limit, page = 0) => {
+const fetchInitialDiscoveryUsers = async (limit = 50, page = 0) => {
   const body = {
     page
   };
 
   if (limit) body.limit = limit;
   try {
-    const response = await api.post('discovery/init/user', body);
+    const response = await api.post(`discovery/init/user?limit=${limit}&page=${page}`, body);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The API endpoint has been changed from a constant string to a template string with limit and page parameters. However, the body of the request also contains these parameters. This could lead to confusion or errors if the server-side code does not handle this correctly. It would be better to stick to one method of passing these parameters - either in the URL or in the body.
```diff
- const response = await api.post(`discovery/init/user?limit=${limit}&page=${page}`, body);
+ const response = await api.post('discovery/init/user', body);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->